src/test_util.js

```diff
@@ -156,4 +157,10 @@ export default class TestUtils {
 
         return true;
     }
+
+    static counterEquals(counter: LocalCounter, value: number): boolean {
+        let valueEqual = counter._backend._counterValues[counter._name] === value;
```

Comment:
I don't think this should be in the public utils. The point of local backend is that you can query it for the current state of counters (which would return some well defined dict-based structure), rather than peeking into implementation details of the counters.